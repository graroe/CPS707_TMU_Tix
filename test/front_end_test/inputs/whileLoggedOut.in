logout
!q
create
add
delete
sell
return
logout