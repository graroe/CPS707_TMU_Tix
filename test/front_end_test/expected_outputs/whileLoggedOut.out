Not currently logged in.
Invalid command. Log in to continue.
Invalid command. Log in to continue.
Invalid command. Log in to continue.
Invalid command. Log in to continue.
Invalid command. Log in to continue.
Invalid command. Log in to continue.
Not currently logged in.